README.md

```diff
@@ -50,6 +50,39 @@ For more advanced configuration, please refer to the `curl` documentation.
 
 ## Development
 
+### System Requirements
+Install the following with your system's package manager
+ - opam
+ - make
+ - clang
+ - patch
+ - curl
+ - libcurl-devel
+
+Helper for opam install on linux:
+`bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"`
```

Comment:
If opam was installed with the package manager, as suggested above, this shouldn't be necessary, so I think we can simplify the instructions slightly.

Reply:
I had to use this as opam wasn't available in the package manager for the OS I built it on (Rocky), but agreed we can leave it out.